src/main/kotlin/Main.kt

```diff
@@ -9,6 +9,7 @@ import net.ayataka.kordis.DiscordClient
 import net.ayataka.kordis.Kordis
 import net.ayataka.kordis.entity.server.enums.ActivityType
 import net.ayataka.kordis.entity.server.enums.UserStatus
+import net.ayataka.kordis.entity.channel.TextChannel
```

Comment:
why tho

Reply:
Ah, probably got in through the rebase merge